### PR TITLE
Add Potato65 Solderable PCB

### PR DESCRIPTION
**Pull request by Aeonstrife**
## Description

Added support for Potato65 Solderable PCB for Potato65 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for the Potato65 solderable PCB.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/buildakb/potato65s/config.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+#define RGB_DI_PIN D5
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/buildakb/potato65s/config.h:56]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/buildakb/potato65s/config.h (hunk 1 above)
Comment (Aeonstrife) [reply keyboards/buildakb/potato65s/config.h]: done 👍

**Review by drashna — APPROVED** [review]

**Comment by Aeonstrife** [comment]
Please can this PR be merged? It has been more than a month since I opened it.

File: keyboards/buildakb/potato65s/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/buildakb/potato65s/rules.mk:20]:
```suggestion
```

File: keyboards/buildakb/potato65s/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/buildakb/potato65s/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/buildakb/potato65s/info.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
+{
+    "keyboard_name": "Potato 65 Solderable",
+    "url": "https://github.com/Aeonstrife/potato65solderable",
+    "maintainer": "Maelkk",
+    "width": 16,
+    "height": 5,
```
Comment (fauxpark) [inline keyboards/buildakb/potato65s/info.json:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**